access/src/main/java/edu/unc/lib/dl/ui/controller/FullRecordController.java

```diff
@@ -216,7 +219,7 @@ public String getFullRecord(String pidString, Model model, HttpServletRequest re
 
 //        if (briefObject.getResourceType().equals(searchSettings.resourceTypeCollection)
 //                || briefObject.getResourceType().equals(searchSettings.resourceTypeFolder)) {
-//            applyContainerSettings(pidString, foxmlView, model, fullObjectView != null);
+//            applyContainerSettings(pidString, principals, foxmlView, model, fullObjectView != null);
```

Comment:
i'm guessing this will be turned back on at some point in the future?

Reply:
It needs to be updated... unless hongs work determines that we don't need this feature anymore